docs/reference/contributing/guidelines/ci.md

```diff
@@ -0,0 +1,25 @@
+## Continuous integration
+
+Continuous integration (CI) means mainly automatic testing for pull requests
+
+### Travis CI
+
+In Mbed OS [Travis CI](https://travis-ci.org/) is used as primary automatic testing and checking run environment.
+
+Travis configuration is located in .travis.yml file in Mbed OS root directory. Mbed OS use public travis so test results are publicly available and there are public [documentation available](https://docs.travis-ci.com/).
+
```

Comment:
> #### What kind of tests jenkins runs?

Should travis also explain each jobs : docs, astyle etc ? like below in the jenkins?

Reply:
Good point!